### DISCUSSION TITLE
pgp encryption for alias.. sender is not signer

### DISCUSSION THREAD

**Discussion by alita-moore** [simple-login/app#759] [Q&A]
basically, I want to send encrypted mail from my proton account through simple login and out an alias. It seems like this isn't possible though because if I generate a key for my proton email, upload it to simple login, and then respond to incoming mail it will error because the signed message is coming from someone other than the owner of the key. 

Is there any way around this?

**Comment by nguyenkims** [comment]
If you want to send an encrypted email to a contact from your alias, you can add a PGP key for this contact. SimpleLogin will encrypt the outgoing email using this key.

![contact-pgp](https://user-images.githubusercontent.com/3167780/151016825-19b3a558-096a-4e9d-93e1-3edbb0a27921.png)

**Reply by alita-moore** [reply]
huh, is there any way to set the value automatically? i.e. public keys are normally attached to encrypted messages

**Reply by nguyenkims** [reply]
This is indeed a good idea, can we create a feature request for this? We also need to make sure to only add the PGP key automatically if the email address contained in the public key corresponds to the contact address.

**Reply by 0xspade** [reply]
like a contact list where you can add PGP to your contacts, so automatically it will encrypt their emails.
